DebugServiceTest failing on Eclipse CI

### Component

JKube Kit

### Task description

### Description

Related to #1753 

I was testing my fix for #1753 on Jenkins CI, I noticed that DebugServiceTest seems to be failing:
```
[ERROR]   DebugServiceTest.debugWithApplicableEntities:212 
Argument(s) are different! Wanted:
silentLogger.info(
    startsWith("Now you can start a Remote debug session by using localhost"),
    <any integer>
);
-> at org.eclipse.jkube.kit.common.KitLogger$SilentLogger.info(KitLogger.java:144)
Actual invocations have different arguments:

[ERROR]   DebugServiceTest.debugWithApplicableEntitiesAndSuspend:234 
Argument(s) are different! Wanted:
silentLogger.info(
    startsWith("Now you can start a Remote debug session by using localhost"),
    <any integer>
);
-> at org.eclipse.jkube.kit.common.KitLogger$SilentLogger.info(KitLogger.java:144)
Actual invocations have different arguments:
```

You can see complete Jenkins job logs: https://ci.eclipse.org/jkube/job/daily-build/613/console . 

 Ever since https://github.com/eclipse/jkube/pull/1878 got merged our [daily jenkins build](https://ci.eclipse.org/jkube/job/daily-build/) has been constantly failing :

```
[ERROR] Failures: 
[ERROR]   DebugServiceTest.debugWithApplicableEntities:212 
[ERROR]   DebugServiceTest.debugWithApplicableEntitiesAndSuspend
```
I'm not able to reproduce this failure on my machine. This seems to only happen on CI.

Looking at these two tests, something looks weird here. DebugServiceTest runs KubernetesMockServer in CRUD mode:
https://github.com/eclipse/jkube/blob/997629eb072c4aa69134dcca93650e6f772c02fe/jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/DebugServiceTest.java#L70

However, these two tests are adding their own expectations 
https://github.com/eclipse/jkube/blob/997629eb072c4aa69134dcca93650e6f772c02fe/jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/DebugServiceTest.java#L199-L203

On CI, it looks like tests are getting stuck on portfoward (which is the API call that's getting mocked in expectations mode)


### Expected Behavior
DebugServiceTest should pass on all environments

### Acceptance Criteria
- [ ] DebugServiceTest should pass in all environments

--- a/jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/PortForwardServicePortOrderTest.java
+++ b/jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/PortForwardServicePortOrderTest.java
@@ -13,6 +13,7 @@
  */
 package org.eclipse.jkube.kit.config.service;
 
+import io.fabric8.kubernetes.client.LocalPortForward;
 import io.fabric8.kubernetes.client.NamespacedKubernetesClient;
 import io.fabric8.kubernetes.client.server.mock.EnableKubernetesMockClient;
 import io.fabric8.kubernetes.client.server.mock.KubernetesMockServer;
@@ -49,9 +50,10 @@ class PortForwardServicePortOrderTest {
         request.complete(r);
         return "";
       }).always();
-    PortForwardService.forwardPortAsync(kubernetesClient, "foo-pod", containerPort, localPort);
-    // When
-    try (final Socket ignored = new Socket(InetAddress.getLocalHost(), localPort)) {
+    try (LocalPortForward localPortForward = PortForwardService.forwardPortAsync(kubernetesClient, "foo-pod", containerPort, localPort);
+         // When
+         final Socket ignored = new Socket(InetAddress.getLocalHost(), localPort)
+    ) {
       // The socket connection triggers the KubernetesClient request to the k8s portforward endpoint
     }
     // Then

